webapp/src/viewModel/cardTree.ts

```diff
@@ -59,12 +64,24 @@ class MutableCardTree implements CardTree {
         const contentBlocks = blocks.filter((block) => contentBlockTypes.includes(block.type as ContentBlockTypes)) as IContentBlock[]
         cardTree.contents = OctoUtils.getBlockOrder(card.contentOrder, contentBlocks)
 
+        cardTree.latestBlock = MutableCardTree.getMostRecentBlock(cardTree)
+
         return cardTree
     }
 
-    // private mutableCopy(): MutableCardTree {
-    //     return MutableCardTree.buildTree(this.card.id, this.allBlocks)!
-    // }
+    public static getMostRecentBlock(cardTree: CardTree): IBlock {
+        let latestBlock: IBlock = cardTree.card
+        cardTree?.allBlocks.forEach((block) => {
```

Comment:
Since it's required, is the `?` needed? It's minor but made me think it can be undefined.
```suggestion
        cardTree.allBlocks.forEach((block) => {
```

Reply:
Done.